skyplane/api/impl/transfer_job.py

```diff
@@ -229,6 +265,11 @@ def verify(self):
 
 @dataclass
 class SyncJob(CopyJob):
+    
+    def __post_init__(self):
+        self.type = "sync"
```

Comment:
Add a "type" class attribute to the dataclass with a default value. That way, it will serialize cleanly.